src/components/Alteration/AlterationSummary.vue

```diff
@@ -235,17 +234,22 @@ export default class AlterationSummary extends Mixins(
 
   // sum of alteration fees
   get alterationFees (): string {
-    if (this.getCurrentFees.filingFees !== null && this.getCurrentFees.futureEffectiveFees !== null) {
-      return `($${(this.getCurrentFees.filingFees + this.getCurrentFees.futureEffectiveFees).toFixed(2)} Fee)`
+    const validFees = this.getCurrentFees.filter(f => f.filingFees !== null && f.futureEffectiveFees !== null)
+    if (validFees.length === 0) {
+      return ''
     }
-    return ''
+    const filingFeesSum = validFees.map(f => f.filingFees).reduce((a, b) => a + b, 0)
```

Comment:
This (and other statements in this method) needs comments to explain what you're doing, please.

Reply:
`filingFeesSum` doesn't describe it enough? Ok I'll add a comment

Reply:
The names are very good. Let's see what the other reviewers think. I was trying to understand the chained map and reduce.

Reply:
Thanks for the extra comments. Further understanding the map and reduce is an exercise left to the reader 😄